package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-subsection-generator",
-  "version": "7.2.0",
+  "version": "7.2.1",
```

Comment:
So, this is very strange. This `layout.js` filename has always (seemingly, since its creation. See: https://github.com/LaunchPadLab/lp-subsection-generator/tree/d45dfe7942420cb567bb9fb5a20eb6b56d0bfddb/template) been incorrect (should have been `Layout.js`). [This PR](https://github.com/LaunchPadLab/lp-subsection-generator/pull/6) changed `routes.js` to `Routes.js`. `layout.js` was never corrected though, even though its inclusion in `Routes.js` was incorrect. There are several version strange issues related to this:
- When I look at the `template` directory in VSCode, the filename seems to be correctly named: https://share.getcloudapp.com/YEuDod0N. 
- When I look in GitHub, it is not: https://share.getcloudapp.com/nOu1km2y.

MacOS is not a case-sensitive filesystem by default even though it seems to honor file case names. I can't explain this.

Reply:
Yeah macOS isn't case-sensitive. It's very possible that you changed it locally and that change wasn't flagged by git. I suspect that because the file is lowercase on my system: https://share.getcloudapp.com/geuOzmlj